### PR TITLE
[Impeller] Make Picture::RenderToTexture antialiased 

### PR DESCRIPTION
**Pull request by dnfield**
Right now the textures aren't getting MSAA and so look aliased when drawing into a scene.

### CODE REVIEW

**Review by jonahwilliams — APPROVED** [review]
LGTM

File: impeller/aiks/picture.cc [hunk 1]
```diff
@@ -52,7 +52,7 @@ std::shared_ptr<Texture> Picture::RenderToTexture(
 
   // This texture isn't host visible, but we might want to add host visible
   // features to Image someday.
-  auto target = RenderTarget::CreateOffscreen(*context.GetContext(), size);
+  auto target = RenderTarget::CreateOffscreenMSAA(*context.GetContext(), size);
```
Comment (bdero) [inline impeller/aiks/picture.cc:55]: This needs a `context->SupportsOffscreenMSAA()` guard for GLES2.

Comment (bdero) [reply impeller/aiks/picture.cc:55]: Hmm, we should add a debug assert for `SupportsOffscreenMSAA` in `CreateOffscreenMSAA` to make this easier to catch.

Comment (dnfield) [reply impeller/aiks/picture.cc:55]: Does that mean for GLES2 we can't get anti-aliased textures?

Comment (dnfield) [reply impeller/aiks/picture.cc]: (and done). Maybe this will be ok because most phones will actually be on Vulkan...

Comment (bdero) [reply impeller/aiks/picture.cc]: Yeah MSAA only works for the onscreen framebuffer in GLES2, so we'd need to employ other techniques if aiming for full GLES2 support in the future.

**Review by bdero — APPROVED** [review]
LGTM!

**Pull request merged**